discord/bot.js

```diff
@@ -1447,15 +1453,18 @@ client.on("interactionCreate", async (interaction) => {
                     components: message.components
                 });
 
-                const success = switchAccount(interaction.user.id, parseInt(accountIndex));
-                if(!success) return await interaction.followUp({
+                if(accountIndex !== "a" && accountIndex !== "d"){
+                    const success = switchAccount(interaction.user.id, parseInt(accountIndex));
+                    if(!success) return await interaction.followUp({
                         embeds: [basicEmbed(s(interaction).error.ACCOUNT_NOT_FOUND)],
                         ephemeral: true
-                });
+                    });
+                }
 
                 let newMessage;
                 switch(customId) {
-                    case "shop": newMessage = await fetchShop(interaction, getUser(interaction.user.id)); break;
+                    case "shop": newMessage = await fetchShop(interaction, getUser(interaction.user.id), interaction.user.id, "d"); break;
+                    case "accessoryshop": newMessage = await fetchShop(interaction, getUser(interaction.user.id), interaction.user.id, "a"); break;
```

Comment:
It wasn't immediately obvious that d = daily and a = accessory. Could you please either rename them to "daily" and "accessory", or make an enum somewhere (probably util.js), or just leave a comment in the `fetchShop()` definition that says something like `// "accessory" can be either "d" for daily, or "a" for accessory`